packages/kernel-5.10/kernel-5.10.spec

```
@@ -291,9 +297,13 @@ mkdir -p %{buildroot}%{_cross_unitdir}/modprobe@neuron.service.d
 install -p -m 0644 %{S:221} %{buildroot}%{_cross_unitdir}/modprobe@neuron.service.d/neuron.conf
 %endif
 
+mkdir -p %{buildroot}%{_cross_datadir}/xfs
+install -p -m 0644 %{S:300} %{buildroot}%{_cross_datadir}/xfs/mkfs.xfs.conf
```

Why not just use the corresponding config file that's already in the xfsprogs package?

```suggestion
mkdir -p %{buildroot}%{_cross_datadir}/xfsprogs/mkfs
ln -s lts_5.10.conf default.conf
```

---

Nice call, thanks! I didn't realize we can use them in that way.